client/webserver/site/src/js/markets.ts

```diff
@@ -2986,6 +2996,7 @@ class BalanceWidget {
     this.quote.cfg = this.dex.assets[quoteID]
     this.updateWallet(this.base)
     this.updateWallet(this.quote)
+    this.setBalanceVisibility(true)
```

Comment:
It takes some searching to verify that `true` is the right argument here. It is, because `setWallet` is only ever called in `setMarket` after a check for connection status. But it's probably safer to do

```Typescript
this.setBalanceVisibility(app().user.exchanges[host].connectionStatus !== ConnectionStatus.Connected)
```

Reply:
updated